### PR TITLE
MUC Room Informations

### PR DESCRIPTION
**Pull request by Neustradamus**
Information -> Informations
gchat_info -> gchat_infos
doShowInfo -> doShowMucInfos
actInfo -> actInfos

### CODE REVIEW

File: options/default.xml [hunk 1]
```diff
@@ -599,17 +599,17 @@ QLineEdit#le_status_text {
                 <chat_share_files type="QKeySequence"></chat_share_files>
                 <chat_templates type="QKeySequence"></chat_templates>
                 <chat_voice type="QKeySequence"></chat_voice>
-                <gchat_info type="QKeySequence"></gchat_info>
                 <gchat_clear type="QKeySequence"></gchat_clear>
                 <gchat_configure type="QKeySequence"></gchat_configure>
                 <gchat_find type="QKeySequence"></gchat_find>
                 <gchat_html_text type="QKeySequence"></gchat_html_text>
                 <gchat_icon type="QKeySequence"></gchat_icon>
+                <gchat_infos type="QKeySequence"></gchat_infos>
```
Comment (Ri0n) [inline options/default.xml:607]: infos? From which word is it?

File: src/groupchatdlg.cpp [hunk 2]
```diff
@@ -1489,7 +1489,7 @@ void GCMainDlg::sendNewTopic(const QMap<LanguageManager::LangId, QString> &topic
     emit aSend(m);
 }
 
-void GCMainDlg::doShowInfo()
+void GCMainDlg::doShowMucInfos()
```
Comment (Ri0n) [inline src/groupchatdlg.cpp:1492]: revert this

File: src/psi_profiles.cpp [hunk 3]
```diff
@@ -525,7 +525,7 @@ void OptionsMigration::lateMigration()
         groupchatToolbar.on = chatToolbar.on;
 
         groupchatToolbar.name = "Groupchat";
-        groupchatToolbar.keys << "gchat_info"
+        groupchatToolbar.keys << "gchat_infos"
```
Comment (Ri0n) [inline src/psi_profiles.cpp:528]: revert this

File: src/psiactionlist.cpp [hunk 4]
```diff
@@ -530,7 +530,7 @@ void PsiActionList::Private::createChat()
 void PsiActionList::Private::createGroupchat()
 {
     {
-        IconAction *actInfo = new IconAction(tr("Information"), QLatin1String("psi/info"), tr("Information"), 0, this);
+        IconAction *actInfos = new IconAction(tr("Informations"), QLatin1String("psi/info"), tr("Informations"), 0, this);
```
Comment (Ri0n) [inline src/psiactionlist.cpp:533]: There is no such a word like "Informations"

**Review by Ri0n — CHANGES_REQUESTED** [review]

**Comment by Ri0n** [comment]
The whole PR is wrong. closing.

File: options/default.xml (hunk 1 above)
Comment (Neustradamus) [reply options/default.xml:607]: yes, there are informations not only one information and there is gchat_info already ;)

File: src/groupchatdlg.cpp (hunk 2 above)
Comment (Neustradamus) [reply src/groupchatdlg.cpp:1492]: It is for MUC Room and there are informations, not only one.

File: src/psi_profiles.cpp (hunk 3 above)
Comment (Neustradamus) [reply src/psi_profiles.cpp:528]: gchat_info already exist, conflict.

File: src/psiactionlist.cpp (hunk 4 above)
Comment (Neustradamus) [reply src/psiactionlist.cpp:533]: Here: https://i.imgur.com/2eTFChM.png

Several informations.

Comment (Ri0n) [reply src/psiactionlist.cpp:533]: There is no plural form of "information" word in English language.

Comment (Neustradamus) [reply src/psiactionlist.cpp:533]: On GitHub:
- https://github.com/search?o=desc&q=informations&s=indexed&type=Code
- https://github.com/search?q=informations&type=code

**Comment by Neustradamus** [comment]
MUC Room informations:
- Room name in the bookmark
- Account currently used
- ID of the mucroom
- Name of the mucroom
- Description of the mucroom

Screenshot:
- https://i.imgur.com/2eTFChM.png

File: src/psiactionlist.cpp (hunk 4 above)
Comment (Ri0n) [reply src/psiactionlist.cpp:533]: I don't care about mistakes of other people.

**Pull request closed without merging**